### DISCUSSION TITLE
Type problems with math example

### DISCUSSION THREAD

**Discussion by skyl** [remarkjs/remark#968] [Q&A] [answered]
Looks like the types are changing; I've tried a few different version combinations:

```
Type 'Plugin<[], Node<Data>, Node<Data>>' is not assignable to type 'Pluggable<any[], Settings>'.
  Type 'Plugin<[], Node<Data>, Node<Data>>' is not assignable to type 'Plugin<any[], Settings>'.
    The 'this' types of each signature are incompatible.
      Property 'attachers' is missing in type 'Processor<Settings>' but required in type 'Processor<void, Node<Data>, void, void>'.ts(2322)
```

```
        <Remark
            remarkPlugins={[
                remarkMath
            ]}
            rehypePlugins={[
                rehypeKatex
            ]}
>{cotent}></Remark>
```

The latest versions have a different, similar error but I was trying to go with what you have in package-lock.json -     `"remark-math": "4.0.0",`

**Comment by ChristianMurphy** [comment, marked as answer]
@skyl sorry you ran into a spot of trouble.
I'm a bit unclear from your description what you are experiencing, for example what packages and what versions are you using?
I tried pluging your example into the `react-markdown` sandbox (which is my best guess at what you are attempting) and don't seem to be able to replicate the issue https://codesandbox.io/s/rehype-katex-dt7qyr
Could you share a sandbox with a reproducible example of the issue you are seeing?

**Reply by skyl** [reply]
Hi thanks, Christian; I ended up the similar `react-markdown` because it worked with more of the types OOTB for some reason. I was moving too fast to really stop and take stock but, I got where I was going - with math, oembeds and syntax highlighting ... https://free2z.com/embeds

We can close this, if "closing" is a thing that can be done with discussions. I feel if I had the time I could get back to some type issues with the example. Thanks though! Sorry for the noise!
